packages/lab/src/carousel/CarouselSlide.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  ElementType,
+  forwardRef,
+  HTMLAttributes,
+  ReactElement,
+  useRef,
+} from "react";
+import { makePrefixer, useAriaAnnouncer } from "@jpmorganchase/uitk-core";
+
+import cx from "classnames";
+import { ButtonBar, ButtonBarProps } from "../buttonbar";
+
+export interface CarouselSlideProps extends HTMLAttributes<HTMLDivElement> {
+  ButtonBar?: ElementType<Partial<ButtonBarProps>>;
+  Media: ReactElement;
```

Comment:
From the code, this is used as a "renderer", maybe consider aligning with `ListItem` from List which is a component